DigitalOcean.API/Clients/ILoadBalancerClient.cs

```diff
@@ -42,11 +42,11 @@ public interface ILoadBalancerClient {
 		/// <summary>
 		/// Add new Forwarding Rule to an LoadBalancer
 		/// </summary>
-		Task AddForwardingRule(int loadBalancerId, ForwardingRules forwardingRules);
+		Task AddForwardingRules(int loadBalancerId, Models.Requests.ForwardingRulesList forwardingRules);
```

Comment:
Both Add and Remove methods actually operate on lists of ForwardingRules. New class to handle this.